test/unit/ldp/http/metadata/SlugParser.test.ts

```diff
@@ -21,8 +21,9 @@ describe('A SlugParser', (): void => {
 
   it('errors if there are multiple slug headers.', async(): Promise<void> => {
     request.headers.slug = [ 'slugA', 'slugB' ];
-    await expect(parser.parse(request, metadata))
-      .rejects.toThrow(new BadRequestHttpError('Request has multiple Slug headers'));
+    const result = parser.parse(request, metadata);
+    await expect(result).rejects.toThrow(BadRequestHttpError);
+    await expect(result).rejects.toThrow('Request has multiple Slug headers');
```

Comment:
I wonder if it wasn't easier to just add a custom matcher to Jest 🙃

Reply:
In hindsight: ...maybe 😅